generator/feature_sorter.cpp

```diff
@@ -368,10 +356,14 @@ bool GenerateFinalFeatures(feature::GenerateInfo const & info, std::string const
         collector(fb);
       }
 
+      LOG(LINFO, ("Writing features' data to", dataFilePath));
+
       // Update bounds with the limit rect corresponding to region borders.
       // Bounds before update can be too big because of big invisible features like a
       // relation that contains an entire country's border.
-      // Borders file may be unavailable when building test mwms.
+      // TODO: Borders file may be unavailable when building test mwms (why?).
+      // Probably its the reason resulting mwm sizes are unpredictable
+      // when building not the whole planet.
```

Comment:
Leave this comment:
/// @todo m_targetDir is a final MWM folder like 220702, so there is NO borders folder inside,
/// and the next function call always returns false.